### PR TITLE
Fix some non-python 3 things

### PR DESCRIPTION
**Pull request by DasFranck**
There's some few things which doesn't works with python3.
Quick-fixed.

### CODE REVIEW

File: sc2reader/engine/plugins/creeptracker.py [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+except:
+    Set = set
 from PIL.Image import open as PIL_open
-from PIL.Image import ANTIALIAS 
-from StringIO import StringIO
+from PIL.Image import ANTIALIAS
+try:
+    from StringIO import StringIO
+except:
+    pass
```
Comment (dsjoerg) [inline sc2reader/engine/plugins/creeptracker.py]: i'm struggling to understand how `pass` can work here.  isnt some kind of StringIO import required in order for the code to be able to work?

**Comment by dsjoerg** [comment]
Hi, reading the code I have only one question which I asked in-line.  In general: does the code pass unit testing with python2 and/or python3?

File: sc2reader/engine/plugins/creeptracker.py (hunk 1 above)
Comment (DasFranck) [inline sc2reader/engine/plugins/creeptracker.py]: My bad, I'm fixing this right now.

**Comment by DasFranck** [comment]
Both of ggtracker and my fork failed on test_replays with Python3 because of an invalid syntax (Python2 print syntax)

```
File "/tmp/sc2reader/test_replays/test_all.py", line 441
  print "DOING {}".format(i)
```

For test_s2gs, it works on my side while it doesn't on the ggtracker version, because of what I've changed.

Nothing seems to have changed for Python2.

**Comment by dsjoerg** [comment]
Hi @DasFranck sorry for the delay.
When you say "Nothing seems to have changed for Python2." you mean that in Python2, the same tests that passed before your change still pass after your change?

**Comment by DasFranck** [comment]
Yes, that's exactly what I meant.

**Comment by dsjoerg** [comment]
@DasFranck so funny we should talk about this, someone just asked on twitter about python3 support and now I can tell them we do!  Thank you.

**Comment by choucavalier** [comment]
good job @DasFranck :+1:

**Pull request merged**